testing/scenario_app/README.md

```diff
@@ -1,85 +1,33 @@
 # Scenario App
 
-This folder contains e2e integration tests for the engine in conjunction with a
-fake dart:ui framework running in JIT or AOT.
+[![GitHub Issues or Pull Requests by label](https://img.shields.io/github/issues/flutter/flutter/e%3A%20scenario-app)](https://github.com/flutter/flutter/issues?q=is%3Aopen+is%3Aissue+label%3A%22e%3A+scenario-app%22)
 
-It intentionally has no dependencies on the Flutter framework or tooling, such
-that it should be buildable as a presubmit or postsubmit to the engine even in
-the face of changes to Dart or dart:ui that require upstream changes in the
-Flutter tooling.
+This package simulates a Flutter app that uses the engine (`dart:ui`) only,
+in conjunction with Android and iOS-specific embedding code that simulates the
+use of the engine in a real app (such as plugins and platform views).
 
-## Adding a New Scenario
+To run the tests, you will need to build the engine with the appropriate
```

Comment:
What is an appropriate configuration?

Reply:
Yeah good point. I deleted this as I think it's safe to assume if you don't know how to build android or ios engine artifacts this directory is not going to help you.